packages/@react-spectrum/s2/src/Popover.tsx

```diff
@@ -79,6 +79,7 @@ let popover = style({
   // Don't be larger than full screen minus 2 * containerPadding
   maxWidth: '[calc(100vw - 24px)]',
   boxSizing: 'border-box',
+  display: 'flex',
```

Comment:
possibly problematic, may affect how children are rendered, or is the child always something predictable?

Reply:
hm, i'm not too sure. mind expanding on it a bit more?

for context, i borrowed this from V3 where we styled it using display: flex. i'm not aware of any known issues with using flex in popover in V3.

Reply:
ah right hm...no i don't think we can make assumptions and say that the child is always predictable. that said, a custom popover/dialog will always be inside of the `PopoverBase` which is where `display: flex` is being applied. if we are concerned about how flex might impact custom popover/dialogs, we could chose not to apply it maybe by detecting the dialog context? but if we do that, then at least for the custom popover stories, you'll see the items appearing before they're actually on the popover

Reply:
yeah, can you bring this up in standup?
we are still in pre-release for s2, but we should make sure this is the way we go, it's sort of API

Reply:
from discussion today, there will always be a component between the Popover and the container children, so this is fine